monai/networks/blocks/fcn.py

```diff
@@ -107,10 +108,10 @@ class FCN(nn.Module):
 
     Args:
         out_channels: number of output channels. Defaults to 1.
-        upsample_mode: The mode of upsampling manipulations, there are two choices:
+        upsample_mode: The mode of upsampling manipulations, there are three choices:
             1) "transpose", uses transposed convolution layers.
-            2) "interpolate", uses standard interpolate way.
-            Using the second mode cannot guarantee the model's reproducibility. Defaults to "transpose".
+            2) "bilinear", uses standard interpolate way.
```

Comment:
line 111 says three options but only 2 of them listed here. I think "standard interpolate way" is not very informative...

Reply:
Thanks, = = a stupid error, I forgot to modify this place, let me correct it.